diff -uraN wireplumber-0.5.10/src/systemd/user/wireplumber.service.in omv-wireplumber-0.5.10/src/systemd/user/wireplumber.service.in
--- wireplumber-0.5.10/src/systemd/user/wireplumber.service.in	2025-05-21 06:32:15.000000000 +0200
+++ omv-wireplumber-0.5.10/src/systemd/user/wireplumber.service.in	2025-05-22 15:27:29.265871225 +0200
@@ -3,6 +3,9 @@
 After=pipewire.service dbus.service
 BindsTo=pipewire.service
 Conflicts=pipewire-media-session.service
+ConditionUser=!root
+ConditionUser=!sddm
+ConditionUser=!@system
 
 [Service]
 LockPersonality=yes
diff -uraN wireplumber-0.5.10/src/systemd/user/wireplumber@.service.in omv-wireplumber-0.5.10/src/systemd/user/wireplumber@.service.in
--- wireplumber-0.5.10/src/systemd/user/wireplumber@.service.in	2025-05-21 06:32:15.000000000 +0200
+++ omv-wireplumber-0.5.10/src/systemd/user/wireplumber@.service.in	2025-05-22 15:28:16.088184679 +0200
@@ -8,6 +8,9 @@
 After=pipewire.service
 BindsTo=pipewire.service
 Conflicts=pipewire-media-session.service
+ConditionUser=!root
+ConditionUser=!sddm
+ConditionUser=!@system
 
 [Service]
 LockPersonality=yes
